python/dgl/data/gindt.py

```diff
@@ -25,6 +25,13 @@ class GINDataset(DGLBuiltinDataset):
     settings. The datasets are ``'MUTAG'``, ``'COLLAB'``, ``'IMDBBINARY'``, ``'IMDBMULTI'``,
     ``'NCI1'``, ``'PROTEINS'``, ``'PTC'``, ``'REDDITBINARY'``, ``'REDDITMULTI5K'``.
 
+    If ``degree_as_nlabel`` is set to ``False``, then ``ndata['label']`` stores the provided node label,
+    otherwise ``ndata['label']`` stores the node in-degrees.
+
+    For graphs that have node attributes, ``ndata['attr']`` stores the node attributes.
+    For graphs that have no attribute, ``ndata['attr']`` stores the corresponding one-hot encoding
```

Comment:
"attribute" -> "attributes"

Reply:
Shouldn't a noun following `no` be in its singular form?

Reply:
Seems that both are ok. Never mind. See 
- https://englishlessonsbrighton.co.uk/followed-singular-plural-noun/
- https://english.stackexchange.com/questions/197798/singular-or-plural-verb-after-no/197818